modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,71 @@
+= Multi-factor Authentication
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Multi-factor authentication provides an additional layer of verification for individual Anypoint Platform users.
+You can require all users in your organization to use multi-factor authentication and exempt specific accounts, or users can manually enable multi-factor authentication in their profile settings.
```

Comment:
I think we should specifically call out that MFA does NOT affect SSO users

Reply:
```suggestion
You can require all users in your organization to use multi-factor authentication and exempt specific accounts, or users can manually enable multi-factor authentication in their profile settings. 

Users who use single sign-on (SSO) cannot configure multi-factor authentication for their accounts. SSO users can configure multi-factor authentication through their identity provider. 
```

Reply:
Is that too much info? Presumably, some IDPs support MFA SSO?

Reply:
Nearly all support.  I might make a slight tweak:

"Users who use single sign-on (SSO) cannot configure multi-factor authentication for their accounts. Instead, SSO users should configure multi-factor authentication through their identity provider."

Reply:
```suggestion
You can require all users in your organization to use multi-factor authentication and exempt specific accounts, or users can manually enable multi-factor authentication in their profile settings.

Users who use single sign-on (SSO) cannot configure multi-factor authentication for their accounts. Instead, SSO users should configure multi-factor authentication through their identity provider.
```